test/test_onnx.py

```diff
@@ -74,6 +74,18 @@ def to_numpy(tensor):
                 else:
                     raise
 
+    def test_new_empty_tensor(self):
+        class Module(torch.nn.Module):
+            def __init__(self):
+                super(Module, self).__init__()
+                self.conv2 = ops.misc.ConvTranspose2d(16, 33, (3, 5))
+
+            def forward(self, input2):
+                return self.conv2(input2)
+
+        input = torch.empty(0)
+        self.run_model(Module(), [(input, )])
```

Comment:
Can you run the model with two different inputs with different shapes, to verify that dynamic shapes work?